__tests__/routes/contentCreator.test.js

```diff
@@ -30,19 +29,19 @@ describe('DELETE /api/creators/:id', () => {
 
   beforeEach(async () => {
     // Insert the fake creator into the database
-    await db.collection('contentcreators').insertOne(fakeCreator);
+    await db.collection('content-creators').insertOne(fakeCreator);
```

Comment:
I am also a little curious on why it is changed to content-creators, but it is not breaking and the code works

Reply:
Maybe PR requester can answer why this is written as kebab case? @MGLarsenSW

Reply:
it was changed to kebab case since it needs to be case sensitive and DB does not accept camelCase when i did the tests. the reason @sofiiagran yours is not breaking is because i change your test to match. also i think we created a file called the exact same so it tried to merged ours together. i have updated your files to match ours since yours was a hardcoded mail called test1@mail.dk, instead of the one i have created called testmail@gmail.com that can be updated in the tests and now it also matches the same email address as all the other tests created by other teams.